### PR TITLE
LPD-31929 - Address ERC gaps in Headless APIs for productDisplayPages

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-31929

### CODE REVIEW

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:sf

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-channel-impl/src/main/java/com/liferay/headless/commerce/admin/channel/internal/resource/v1_0/ProductDisplayPageResourceImpl.java [hunk 1]
```diff
@@ -129,12 +131,25 @@ public ProductDisplayPage patchProductDisplayPage(
 			throw new NoSuchCPDisplayLayoutException();
 		}
 
+		long productId = GetterUtil.getLong(productDisplayPage.getProductId());
+
+		if (productId == 0) {
```
[the remark this answers is not in the document]
Comment (smottal) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-channel-impl/src/main/java/com/liferay/headless/commerce/admin/channel/internal/resource/v1_0/ProductDisplayPageResourceImpl.java:136]: it's already verified inside the service layer

**Pull request closed without merging**